ReactWindows/ChakraBridge/JsModulesUnbundle.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <stdint.h>
+
+struct JsModulesUnbundleModule sealed
+{
+public:
+	wchar_t* source;
+	wchar_t* sourceUrl;
+	~JsModulesUnbundleModule()
+	{
+		delete[] source;
+		delete[] sourceUrl;
+	}
+};
+
+class JsModulesUnbundle
+{
+public:
+    JsModulesUnbundle(const wchar_t* sourcePath);
+	~JsModulesUnbundle();
+	virtual JsModulesUnbundleModule* GetModule(uint32_t index);
+	virtual wchar_t* GetStartupCode();
```

Comment:
Same thing, would it be more useful to have an error code returned and an out param for the `wchar_t`?